src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptions/OcrValidationException.java

```diff
@@ -1,7 +1,21 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.exceptions;
 
 public class OcrValidationException extends InvalidMetafileException {
+
+    //might contain sensitive data
+    private String detailMessage;
+
     public OcrValidationException(String message) {
         super(message);
+        this.detailMessage = message;
+    }
+
+    public OcrValidationException(String message, String detailMessage) {
+        super(message);
+        this.detailMessage = detailMessage;
+    }
+
+    public String getDetailMessage() {
+        return detailMessage;
     }
```

Comment:
Looks like this is always used and never the 'normal' message.
Can we just put the detailed message to message field?

Reply:
we do not want to log that is the reason

Reply:
log.error("exception ", ex) will print the message